src/applications/personalization/rated-disabilities/components/RatedDisabilityListItem.jsx

```diff
@@ -29,7 +29,11 @@ const RatedDisabilityListItem = ({ ratedDisability }) => {
         <div className="vads-l-col--12 vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
           <p className="vads-u-margin--0">
             {decisionText === 'Service Connected' ? (
-              <i className="fas fa-medal vads-u-margin-right--0p5" />
+              <i
+                className="fas fa-medal vads-u-margin-right--0p5"
+                aria-hidden="true"
```

Comment:
Adding `aria-hidden` for decorative icon accessibility. More info [here](https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility) and on [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3683)

Reply:
Turns out this doesn't matter much, since the icon is going away in a separate PR ._.